pkgs/development/python-modules/analytics-python/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, monotonic
+, backoff
+, python-dateutil
+}:
+
+buildPythonPackage rec {
+  pname = "analytics-python";
+  version = "1.4.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "plFBq25H2zlvW8Vwix25P/mpmILYH+gIIor9Xrtt/l8=";
```

```suggestion
    hash = "sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=";
```
Could you please use SRI hash instead?
https://nixos.org/manual/nixpkgs/stable/#fetchurl

---

I just copied the previous hash and put it in hash with "sha256-" - is this okay?